### ISSUE TITLE
test suite failures on Debian Bullseye

### ISSUE THREAD

**Issue by nwalfield** [rpm-software-management/rpm#2264]
I realize @pmatilai is aware of these, but I wanted to document them.

```
$ git describe
rpm-4.17.0-alpha-659-g40d66336a
$ make check VERBOSE=1
...
## ----------------------- ##
## rpm 4.18.90 test suite. ##
## ----------------------- ##

  1: Running tests for malformed OpenPGP packages    ok
  2: Running tests for computing OpenPGP fingerprints ok
  5: rpm library version                             ok
  7: rpm --querytags                                 ok
  3: rpm --version                                   ok
  4: rpmbuild --version                              ok
  6: rpm --showrc                                    ok
  8: rpm2cpio                                        ok
 10: rpmvercmp(1.0, 1.0) = 0                         ok
 11: rpmvercmp(1.0, 2.0) = -1                        ok
 13: rpmvercmp(2.0.1, 2.0.1) = 0                     ok
 12: rpmvercmp(2.0, 1.0) = 1                         ok
 14: rpmvercmp(2.0, 2.0.1) = -1                      ok
  9: rpm2archive                                     ok
 15: rpmvercmp(2.0.1, 2.0) = 1                       ok
 16: rpmvercmp(2.0.1a, 2.0.1a) = 0                   ok
 17: rpmvercmp(2.0.1a, 2.0.1) = 1                    ok
 18: rpmvercmp(2.0.1, 2.0.1a) = -1                   ok
 19: rpmvercmp(5.5p1, 5.5p1) = 0                     ok
 20: rpmvercmp(5.5p1, 5.5p2) = -1                    ok
 21: rpmvercmp(5.5p2, 5.5p1) = 1                     ok
 22: rpmvercmp(5.5p10, 5.5p10) = 0                   ok
 23: rpmvercmp(5.5p1, 5.5p10) = -1                   ok
 24: rpmvercmp(5.5p10, 5.5p1) = 1                    ok
 25: rpmvercmp(10xyz, 10.1xyz) = -1                  ok
 26: rpmvercmp(10.1xyz, 10xyz) = 1                   ok
 27: rpmvercmp(xyz10, xyz10) = 0                     ok
 28: rpmvercmp(xyz10, xyz10.1) = -1                  ok
 29: rpmvercmp(xyz10.1, xyz10) = 1                   ok
 30: rpmvercmp(xyz.4, xyz.4) = 0                     ok
 31: rpmvercmp(xyz.4, 8) = -1                        ok
 33: rpmvercmp(xyz.4, 2) = -1                        ok
 32: rpmvercmp(8, xyz.4) = 1                         ok
 35: rpmvercmp(5.5p2, 5.6p1) = -1                    ok
 34: rpmvercmp(2, xyz.4) = 1                         ok
 36: rpmvercmp(5.6p1, 5.5p2) = 1                     ok
 38: rpmvercmp(6.5p1, 5.6p1) = 1                     ok
 37: rpmvercmp(5.6p1, 6.5p1) = -1                    ok
 39: rpmvercmp(6.0.rc1, 6.0) = 1                     ok
 40: rpmvercmp(6.0, 6.0.rc1) = -1                    ok
 41: rpmvercmp(10b2, 10a1) = 1                       ok
 42: rpmvercmp(10a2, 10b2) = -1                      ok
 43: rpmvercmp(1.0aa, 1.0aa) = 0                     ok
 44: rpmvercmp(1.0a, 1.0aa) = -1                     ok
 45: rpmvercmp(1.0aa, 1.0a) = 1                      ok
 46: rpmvercmp(10.0001, 10.0001) = 0                 ok
 47: rpmvercmp(10.0001, 10.1) = 0                    ok
 48: rpmvercmp(10.1, 10.0001) = 0                    ok
 49: rpmvercmp(10.0001, 10.0039) = -1                ok
 50: rpmvercmp(10.0039, 10.0001) = 1                 ok
 51: rpmvercmp(4.999.9, 5.0) = -1                    ok
 52: rpmvercmp(5.0, 4.999.9) = 1                     ok
 53: rpmvercmp(20101121, 20101121) = 0               ok
 55: rpmvercmp(20101122, 20101121) = 1               ok
 54: rpmvercmp(20101121, 20101122) = -1              ok
 56: rpmvercmp(2_0, 2_0) = 0                         ok
 57: rpmvercmp(2.0, 2_0) = 0                         ok
 58: rpmvercmp(2_0, 2.0) = 0                         ok
 59: rpmvercmp(a, a) = 0                             ok
 60: rpmvercmp(a+, a+) = 0                           ok
 63: rpmvercmp(+a, +a) = 0                           ok
 62: rpmvercmp(a_, a+) = 0                           ok
 61: rpmvercmp(a+, a_) = 0                           ok
 64: rpmvercmp(+a, _a) = 0                           ok
 66: rpmvercmp(+_, +_) = 0                           ok
 65: rpmvercmp(_a, +a) = 0                           ok
 67: rpmvercmp(_+, +_) = 0                           ok
 68: rpmvercmp(_+, _+) = 0                           ok
 69: rpmvercmp(+, _) = 0                             ok
 70: rpmvercmp(_, +) = 0                             ok
 71: rpmvercmp(1.0~rc1, 1.0~rc1) = 0                 ok
 72: rpmvercmp(1.0~rc1, 1.0) = -1                    ok
 74: rpmvercmp(1.0~rc1, 1.0~rc2) = -1                ok
 75: rpmvercmp(1.0~rc2, 1.0~rc1) = 1                 ok
 73: rpmvercmp(1.0, 1.0~rc1) = 1                     ok
 76: rpmvercmp(1.0~rc1~git123, 1.0~rc1~git123) = 0   ok
 77: rpmvercmp(1.0~rc1~git123, 1.0~rc1) = -1         ok
 78: rpmvercmp(1.0~rc1, 1.0~rc1~git123) = 1          ok
 79: rpmvercmp(1.0^, 1.0^) = 0                       ok
 80: rpmvercmp(1.0^, 1.0) = 1                        ok
 81: rpmvercmp(1.0, 1.0^) = -1                       ok
 82: rpmvercmp(1.0^git1, 1.0^git1) = 0               ok
 83: rpmvercmp(1.0^git1, 1.0) = 1                    ok
 84: rpmvercmp(1.0, 1.0^git1) = -1                   ok
 85: rpmvercmp(1.0^git1, 1.0^git2) = -1              ok
 86: rpmvercmp(1.0^git2, 1.0^git1) = 1               ok
 87: rpmvercmp(1.0^git1, 1.01) = -1                  ok
 88: rpmvercmp(1.01, 1.0^git1) = 1                   ok
 89: rpmvercmp(1.0^20160101, 1.0^20160101) = 0       ok
 90: rpmvercmp(1.0^20160101, 1.0.1) = -1             ok
 91: rpmvercmp(1.0.1, 1.0^20160101) = 1              ok
 93: rpmvercmp(1.0^20160102, 1.0^20160101^git1) = 1  ok
 92: rpmvercmp(1.0^20160101^git1, 1.0^20160101^git1) = 0 ok
 94: rpmvercmp(1.0^20160101^git1, 1.0^20160102) = -1 ok
 95: rpmvercmp(1.0~rc1^git1, 1.0~rc1^git1) = 0       ok
 96: rpmvercmp(1.0~rc1^git1, 1.0~rc1) = 1            ok
 97: rpmvercmp(1.0~rc1, 1.0~rc1^git1) = -1           ok
 98: rpmvercmp(1.0^git1~pre, 1.0^git1~pre) = 0       ok
 99: rpmvercmp(1.0^git1, 1.0^git1~pre) = 1           ok
101: simple rpm --eval                               ok
100: rpmvercmp(1.0^git1~pre, 1.0^git1) = -1          ok
102: invalid rpm --eval                              ok
103: invalid rpm --define                            ok
105: simple true conditional rpm --eval              ok
106: simple false conditional rpm --eval             ok
107: nested macro in name                            ok
104: rpm --undefine                                  ok
109: parametrized macro 2                            ok
110: parametrized macro 3                            ok
111: parametrized macro 4                            ok
112: parametrized macro 5                            ok
113: parametrized macro 6                            ok
114: uncompress macro 1                              ok
117: basename macro                                  ok
118: shrink macro                                    ok
119: suffix macro                                    ok
120: url2path macro                                  ok
121: macrobody macro                                 ok
123: macro arguments                                 ok
124: string functions                                ok
125: expr macro 1                                    ok
127: ternary expressions                             ok
128: short circuiting                                ok
129: shell expansion                                 ok
130: expression expansion 1                          ok
132: expression version comparison                   ok
133: simple lua --eval                               ok
134: lua macro arguments                             ok
131: expression expansion 2                          ok
126: expr macro 2                                    ok
135: lua macros table                                ok
137: lua rpm extensions 1                            ok
136: lua macros recursion                            ok
138: lua rpm extensions 2                            ok
139: lua rpm isdefined                               ok
141: lua posix extensions                            ok
140: lua rpm split/unsplitargs                       ok
142: lua script exit behavior                        ok
143: lua script redirect2null                        ok
122: builtin macro arguments                         ok
146: lua rpm version objects                         ok
148: lua functions in expressions                    ok
149: lua hooks                                       ok
150: %define + %undefine in nested levels 1          ok
151: %define + %undefine in nested levels 2          ok
152: %define + %undefine in nested levels 3          expected failure (rpmmacro.at:899)
153: %define + %undefine in nested levels 4          expected failure (rpmmacro.at:917)
154: %define in conditional macro                    ok
156: %exists macro                                   ok
155: %verbose macro                                  ok
157: %shescape macro                                 ok
158: macro with a line starting by "{"               ok
160: macro file errors                               ok
159: macro with %if-%endif                           ok
161: macro comments                                  ok
162: macro comments 2                                ok
163: macro file empty lines                          ok
164: macro traceback                                 ok
165: no space left on stdout                         ok
166: rpmlua                                          ok
108: parametrized macro 1                            ok
116: uncompress 2                                    ok
115: uncompress 1                                    ok
147: lua rpm io stream                               ok
144: lua library path                                ok
145: lua auto-print                                  ok
167: module import                                   skipped (rpmpython.at:6)
168: macro manipulation                              skipped (rpmpython.at:12)
169: basic rpmio                                     skipped (rpmpython.at:35)
170: spec parse 1                                    skipped (rpmpython.at:70)
171: spec parse 2                                    skipped (rpmpython.at:85)
173: non-utf8 data in header                         skipped (rpmpython.at:114)
172: basic header manipulation                       skipped (rpmpython.at:90)
174: invalid header data                             skipped (rpmpython.at:133)
175: labelCompare                                    skipped (rpmpython.at:152)
176: vfyflags API                                    skipped (rpmpython.at:192)
181: add package to transaction                      skipped (rpmpython.at:255)
180: reading a signed package file 2                 skipped (rpmpython.at:237)
178: reading a package file                          skipped (rpmpython.at:214)
177: vfylevel API                                    skipped (rpmpython.at:203)
182: add erasure to transaction                      skipped (rpmpython.at:267)
179: reading a signed package file 1                 skipped (rpmpython.at:222)
183: add restore to transaction                      skipped (rpmpython.at:289)
184: add bogus package to transaction 1              skipped (rpmpython.at:311)
185: add bogus package to transaction 2              skipped (rpmpython.at:325)
187: transaction callback 1                          skipped (rpmpython.at:360)
186: transaction element userdata                    skipped (rpmpython.at:346)
190: dependency sets 1                               skipped (rpmpython.at:563)
189: database cookies                                skipped (rpmpython.at:504)
188: database iterators                              skipped (rpmpython.at:410)
191: dependency sets 2                               skipped (rpmpython.at:580)
192: dependency sets 3                               skipped (rpmpython.at:590)
194: file sets 1                                     skipped (rpmpython.at:627)
193: file info sets 1                                skipped (rpmpython.at:610)
197: archive 1                                       skipped (rpmpython.at:679)
195: string pool 1                                   skipped (rpmpython.at:646)
196: string pool 2                                   skipped (rpmpython.at:661)
198: header unload                                   skipped (rpmpython.at:699)
199: version objects 1                               skipped (rpmpython.at:716)
200: version objects 2                               skipped (rpmpython.at:747)
201: provide - require pairs                         skipped (rpmdepmatch.at:4)
203: rpm --qf -p *.src.rpm                           ok
204: rpm -qp <glob>                                  ok
202: rpm --qf -p *.i386.rpm                          ok
205: rpm -qp <glob fallback>                         ok
206: rpm -qp <notfound>                              ok
207: rpm -qp <glob notfound>                         ok
208: rpm -ql -p *.src.rpm                            ok
210: rpm -qp --dump                                  ok
209: rpm -ql multiple *.rpm                          ok
211: rpmspec -q                                      ok
212: rpm -ql -p *.i386.rpm                           ok
213: rpm -qp <manifest>                              ok
214: rpm -q --scripts -p *.i386.rpm                  ok
215: rpm -q on installed package                     ok
216: rpm -qf                                         ok
219: integer array query                             ok
220: formatted filesbypkg query                      ok
221: hex formatted integer array extension query     ok
222: base64 extension query                          ok
218: rpm -q --path on non-installed file             ok
217: rpm -qf on non-installed file                   ok
223: pgpsig extension query                          ok
224: integer array perms format query                ok
228: empty string array extension query              ok
225: string array query                              ok
226: empty string array query                        ok
227: empty string array extension array format       ok
229: different sizes arrays query 1                  ok
230: different sizes arrays query 2                  ok
231: different sizes arrays query 3                  ok
232: different sizes arrays query 4                  ok
233: non-existent string tag                         ok
235: conditional queryformat                         ok
234: non-existent integer tag query                  ok
236: invalid tag query                               ok
237: invalid data for format query                   ok
238: invalid format width query                      ok
239: xml format                                      ok
244: rpmspec --query Requires                        ok
245: rpmspec --query Requires(,)                     ok
246: rpmspec --query Requires(;)                     ok
247: rpmspec --query Requires(junk)                  ok
248: rpmspec --query Requires(pre,junk)              ok
249: rpmspec --query Requires(junk,pre)              ok
251: rpmspec --query Requires(pre,)                  ok
250: rpmspec --query Requires(,pre)                  ok
252: rpmspec --query Requires(pre,,postun)           ok
253: rpmspec --query Requires(pre,,junk)             ok
254: rpmspec --query Requires(;pre)                  ok
255: rpmspec --query Requires(pre;)                  ok
257: rpmspec --query Requires(pre;junk)              ok
256: rpmspec --query Requires(pre;postun)            ok
258: rpmspec --parse                                 ok
259: rpmspec --srpm provides                         ok
240: query file attribute filtering                  ok
242: incomplete escape sequence for format query     ok
243: query format iteration                          ok
241: formatting name humansi, humaniec               ok
260: rpmspec --parse                                 ok
262: rpmkeys -K <unsigned 2> verifylevel             ok
261: rpmkeys -K <unsigned 1> verifylevel             ok
263: rpmkeys -K <signed 1> verifylevel               ok
264: rpmkeys -K <signed 2> verifylevel               ok
266: rpmkeys -Kv <unsigned> 1                        ok
267: rpmkeys -K no space left on stdout              ok
268: rpmkeys -Kv <reconstructed> 1                   ok
265: rpmkeys -K <signed 3> verifylevel               ok
269: rpmkeys -Kv <corrupted unsigned> 1              ok
276: rpmkeys --import, signed with a good subkey     skipped (rpmsigdig.at:319)
277: rpmkeys --import, signed with an expired subkey skipped (rpmsigdig.at:384)
278: rpmkeys --import, signed with a revoked subkey  skipped (rpmsigdig.at:451)
270: rpmkeys -Kv <corrupted unsigned> 2              ok
271: rpmkeys -Kv <corrupted unsigned> 3              ok
272: rpmkeys -Kv <corrupted unsigned> 4              ok
275: rpmkeys --import rsa (fs)                       ok
274: rpmkeys --import rsa (rpmdb)                    ok
279: rpmkeys --import invalid keys                   ok
273: rpmkeys -Kv <unsigned> 2                        ok
280: rpmkeys --import different-creation-times       ok
281: rpmkeys type confusion                          ok
282: rpmkeys -K <signed> 1                           ok
283: rpmkeys -Kv <signed> 1                          ok
284: rpmkeys -Kv <corrupted signed> 1                ok
285: rpmkeys -Kv <corrupted signed> 2                ok
286: rpmkeys -Kv <corrupted signed> 3                ok
287: rpmkeys -Kv <corrupted signed> 4                ok
289: rpmsign --delsign                               ok
290: SIGPIPE from install scriptlet                  ok
296: directory replaced with an invalid directory symlink skipped (rpmverify.at:94)
288: rpmsign --addsign                               ok
291: rpmlog error handling                           ok
293: files with no problems                          ok
292: dependency problems                             ok
294: files with no problems in verbose mode          ok
295: directory replaced with a directory symlink     ok
297: verify from db, with problems present           ok
298: verify from package, with problems present      ok
300: verifyscript failure                            ok
299: verify file attribute filtering                 ok
307: verify empty/no capabilities 1                  skipped (rpmverify.at:582)
308: verify empty/no capabilities 2                  skipped (rpmverify.at:601)
301: verifyscript success                            ok
304: minimize writes (hardlinks)                     ok
306: minimize writes (SUID files)                    ok
302: shared file timestamp behavior                  ok
309: rpm --restore                                   ok
310: rpm --initdb                                    ok
303: minimize writes (files)                         ok
305: minimize writes (symlinks)                      ok
311: rpm -qa 1                                       ok
312: rpm -qa 2                                       ok
313: rpm -qa 3                                       ok
314: rpm -q foo                                      ok
315: rpm -q foo-                                     ok
317: rpm -q --querybynumber                          ok
316: rpmdb header numbering                          ok
318: rpm -i *.noarch.rpm                             ok
319: rpm -U --replacepkgs 1                          ok
320: rpm -U --replacepkgs 2                          ok
322: rpm -i --relocate=.. *.i386.rpm                 ok
323: rpm -i --relocate=.. *.ppc64.rpm                ok
321: rpm --reinstall 1                               ok
324: rpmdb --rebuilddb                               FAILED (rpmdb.at:290)
325: rpmdb --rebuilddb and verify empty database     FAILED (rpmdb.at:314)
326: rpm -U and verify status                        ok
327: rpm -U with _install_lang and verify status     ok
329: rpm -U <manifest>                               ok
330: rpm -U <manifest glob>                          ok
331: rpm -U <manifest glob fallback>                 ok
332: rpm -U <manifest notfound 1>                    ok
333: rpm -U <manifest notfound 2>                    ok
328: rpmdb query special chars                       ok
334: rpm -U <manifest notfound 3>                    ok
335: rpm -U <manifest notfound 4>                    ok
336: rpm -U <notfound>                               ok
337: rpm -U <notfound 2>                             ok
338: rpm -U <unsigned 1>                             ok
339: rpm -U <unsigned 2>                             ok
340: rpm -U <corrupted unsigned 1>                   ok
341: rpm -U <corrupted unsigned 2>                   ok
342: rpm -U <signed nokey 1>                         ok
343: rpm -U <signed nokey 2>                         ok
344: rpm -U <signed 1>                               ok
346: rpm -U <corrupted signed 1>                     ok
347: rpm -U <corrupted signed 2>                     ok
345: rpm -U <signed 2>                               ok
348: rpm -U <corrupted signed 3>                     ok
349: rpm -U <glob>                                   ok
350: rpm -U <glob notfound>                          ok
351: rpm -U <glob fallback>                          ok
354: rpm -i --nodb                                   ok
355: rpm -i --justdb                                 ok
353: rpm -i <overridden time keeps ticking>          ok
352: rpm -i <overridden time>                        ok
356: rpm -U *.src.rpm                                ok
357: rpm -i *.src.rpm                                ok
358: rpm -i <nonexistent file>                       ok
359: rpm -i --nomanifest <garbage text file>         ok
360: rpm -i <garbage text file                       ok
364: rpm -U with invalid --relocate                  ok
365: rpm -U --badreloc with invalid --relocate       ok
363: rpm -U obsoleted packages                       ok
366: rpm -i relocatable package                      FAILED (rpmi.at:1069)
367: rpm -i with/without --excludedocs               ok
361: rpm upgrade/downgrade                           ok
362: rpm upgrade/downgrade epoch                     ok
368: rpm -i --excludeartifacts                       ok
371: rpm -U filesystem                               ok
372: rpm -e and verify files removed                 ok
369: rpm -U <suicidal>                               ok
373: rpm reinstall with shared files                 ok
370: rpm -i hardlinks                                ok
374: rpm -e and shared files removed 1.1             ok
375: rpm -e and shared files removed 1.2             ok
376: rpm -e and verify colored files removed 1.1     ok
377: rpm -e and verify colored files removed 1.2     ok
378: rpm -e and verify colored files removed 2.1     expected failure (rpme.at:216)
379: rpm -e and verify colored files removed 2.2     expected failure (rpme.at:241)
381: rpm -e and verify netshared files not removed   ok
382: basic install/erase order 1                     ok
383: basic install/erase order 2                     ok
384: basic install/erase order 3                     ok
385: rpmbuild -ba *.spec                             ok
380: rpm -e and verify conflicting files removed 1   ok
387: rpmbuild -ba autosetup                          ok
386: rpmbuild -b steps                               ok
388: rpmbuild -ba autopatch                          ok
389: rpmbuild --rebuild                              ok
391: rpmbuild -tb <tar with bad spec>                ok
392: rpmbuild -ts <spec>                             ok
390: rpmbuild --short-circuit -bl                    ok
393: rpmbuild package with weird filenames           ok
394: rpmbuild package with illegal filenames         ok
395: rpmbuild -tb                                    ok
396: rpmbuild scriptlet -f                           ok
397: rpmbuild %attr and %defattr                     ok
398: rpmbuild unpackaged files                       ok
399: rpmbuild unpackaged directories                 expected failure (rpmbuild.at:420)
402: rpmbuild bad escape                             ok
400: rpmbuild glob                                   ok
401: rpmbuild glob escape                            ok
403: rpmbuild bad quote                              ok
404: rpmbuild prefixpostfix                          ok
406: Build requires                                  ok
405: Weak and reverse requires                       ok
407: Dependency generation 1                         FAILED (rpmbuild.at:648)
413: rpmspec query 1                                 ok
408: Dependency generation 2                         FAILED (rpmbuild.at:690)
415: rpmbuild debuginfo minisymtab                   skipped (rpmbuild.at:860)
416: rpmbuild debuginfo minisymtab strip -g          skipped (rpmbuild.at:898)
417: rpmbuild debuginfo dwz                          skipped (rpmbuild.at:938)
418: rpmbuild no debuginfo                           skipped (rpmbuild.at:1025)
419: rpmbuild debuginfo dwz unique debug names       skipped (rpmbuild.at:1059)
420: rpmbuild implicit suid binary                   skipped (rpmbuild.at:1146)
421: rpmbuild debuginfo gdb index included           skipped (rpmbuild.at:1178)
422: rpmbuild debuginfo no gdb index included        skipped (rpmbuild.at:1205)
423: rpmbuild debuginfo -g3 .debug_macro             skipped (rpmbuild.at:1231)
424: rpmbuild debuginfo unique debug src dir         skipped (rpmbuild.at:1262)
425: rpmbuild debuginfo no unique debug src dir      skipped (rpmbuild.at:1299)
426: rpmbuild debugsource                            skipped (rpmbuild.at:1333)
427: rpmbuild debugsource debugsourcefiles.list path skipped (rpmbuild.at:1365)
428: rpmbuild debuginfo subpackages single           skipped (rpmbuild.at:1390)
429: rpmbuild debuginfo subpackages multiple         skipped (rpmbuild.at:1429)
430: rpmbuild debuginfo subpackages multiple unique  skipped (rpmbuild.at:1510)
409: Dependency generation 3                         FAILED (rpmbuild.at:705)
431: rpmbuild debuginfo subpackages multiple unique debugsource skipped (rpmbuild.at:1591)
432: rpmbuild debuginfo subpackages multiple excluded skipped (rpmbuild.at:1675)
433: rpmbuild debuginfo subpackages multiple excluded skipped (rpmbuild.at:1736)
410: Dependency generation 4                         FAILED (rpmbuild.at:726)
411: Dependency generation 5                         ok
412: elf dependencies                                ok
434: dynamic build requires rpmbuild -bs             ok
437: rpmbuild -br success                            ok
436: rpmbuild -br success                            ok
435: rpmbuild -br                                    ok
438: rpmbuild -bd with errors                        ok
414: rpmbuild archive sanity                         ok
439: rpmbuild -bd *.spec                             ok
440: rpmbuild -ba                                    ok
443: rpmbuild minimal spec                           ok
442: rpmbuild -ba missing source                     ok
441: rpmbuild -br --nodeps                           ok
444: rpmbuild missing files                          ok
445: rpmbuild missing doc                            ok
446: spec conditionals                               ok
450: rpmbuild buildid none                           skipped (rpmbuildid.at:25)
451: rpmbuild buildid alldebug                       skipped (rpmbuildid.at:56)
452: rpmbuild buildid alldebug unique debug names    skipped (rpmbuildid.at:145)
453: rpmbuild buildid separate                       skipped (rpmbuildid.at:233)
454: rpmbuild buildid separate unique debug names    skipped (rpmbuildid.at:321)
455: rpmbuild buildid compat                         skipped (rpmbuildid.at:408)
456: rpmbuild buildid compat unique debug names      skipped (rpmbuildid.at:509)
457: rpmbuild buildid duplicate alldebug             skipped (rpmbuildid.at:610)
458: rpmbuild buildid hardlink alldebug              skipped (rpmbuildid.at:675)
459: rpmbuild buildid duplicate separate             skipped (rpmbuildid.at:737)
460: rpmbuild buildid hardlink separate              skipped (rpmbuildid.at:799)
461: rpmbuild buildid duplicate compat               skipped (rpmbuildid.at:858)
462: rpmbuild buildid hardlink compat                skipped (rpmbuildid.at:932)
463: rpmbuild buildid recompute                      skipped (rpmbuildid.at:1002)
464: rpmbuild buildid unique r1 r2                   skipped (rpmbuildid.at:1113)
465: rpmbuild buildid non-unique r1 r2               skipped (rpmbuildid.at:1158)
466: rpmbuild buildid attrs                          skipped (rpmbuildid.at:1207)
467: rpmbuild buildid config attrs                   skipped (rpmbuildid.at:1238)
447: bcond macro                                     ok
448: rpmbuild spec tag                               ok
449: %sources and %patches                           ok
469: basic scripts and arguments                     ok
471: basic file trigger scripts                      ok
470: basic trigger scripts and arguments             ok
472: basic file triggers 2                           ok
473: unversioned requires                            ok
468: basic script failures 1                         ok
474: unsatisfied versioned require                   ok
475: satisfied versioned require                     ok
477: install and verify self-conflicting package     ok
476: versioned conflicts                             ok
478: explicit file conflicts                         ok
479: erase to break dependencies                     ok
480: erase to break colored file dependency          ok
481: erase on wrong-colored file dependency          ok
482: unsatisfied WITH requires                       ok
483: satisfied WITH requires                         ok
484: unsatisfied WITHOUT requires                    ok
485: satisfied WITHOUT requires                      ok
486: AND requires                                    ok
487: OR requires                                     ok
488: IF requires                                     ok
489: IF-ELSE requires                                ok
490: nested AND-OR requires                          ok
491: nested AND-IF requires                          ok
492: install to break installed rich dependency      ok
493: erase to break installed rich dependency        ok
494: packages with file conflicts                    ok
495: shareable files                                 ok
496: non-conflicting identical basenames             FAILED (rpmconflict.at:79)
497: conflicting identical basenames                 FAILED (rpmconflict.at:95)
498: multilib elf conflict, prefer 64bit 1           ok
499: multilib elf conflict, prefer 64bit 2           ok
500: multilib elf conflict, prefer 64bit 3           ok
501: multilib elf conflict, prefer 32bit 1           ok
502: multilib elf conflict, prefer 32bit 2           ok
504: multilib elf vs non-elf file conflict 1         ok
503: multilib elf conflict, prefer 32bit 3           ok
505: multilib elf vs non-elf file conflict 2         ok
506: multilib elf vs non-elf file conflict 3         ok
507: replacing directory with symlink                ok
508: replacing symlink with directory                ok
509: real file with shared ghost                     ok
510: upgrade to/from regular file                    ok
512: upgrade broken link to broken link              ok
511: upgrade regular file to/from link               ok
513: upgrade file link to file link                  ok
514: upgrade directory link to directory link        ok
515: upgrade regular file to directory               ok
516: upgrade broken link to directory                ok
517: upgrade file link to directory                  ok
518: upgrade directory link to directory             ok
519: upgrade empty directory to empty directory      ok
520: upgrade empty directory to regular file         ok
521: upgrade locally symlinked directory             ok
522: upgrade invalid locally symlinked directory     ok
523: upgrade empty directory to broken link          ok
525: upgrade empty directory to file link with pretrans ok
524: upgrade empty directory to file link            ok
527: config(noreplace) file install/upgrade/erase    ok
528: upgrade changing config                         ok
529: upgrade changing config(noreplace)              ok
526: config file install/upgrade/erase               ok
530: upgrade changing config(noreplace)              ok
531: upgrade shared config                           ok
533: upgrade shared config(noreplace)                ok
534: ghost config                                    ok
532: upgrade changing shared config                  ok
536: changing config symlinks                        ok
537: upgrade unchanged config(noreplace) links       ok
535: config symlinks                                 ok
538: upgrade changing config(noreplace) links        ok
539: upgrade unchanged shared config links           ok
541: upgrade changing shared config(noreplace) links ok
540: upgrade changing shared config links            ok
542: upgrade config to/from non-config               ok
544: upgrade modified config link to config          ok
543: upgrade config to/from config link              ok
545: upgrade config to directory                     ok

## ------------- ##
## Test results. ##
## ------------- ##

ERROR: 467 tests were run,
14 failed (5 expected failures).
78 tests were skipped.
## ------------------------- ##
## rpmtests.log was created. ##
## ------------------------- ##

Please send `rpmtests.log' and all information you think might help:

   To: <http://rpm.org>
   Subject: [rpm 4.18.90] rpmtests: 324 325 366 407 408 409 410 496 497 failed

You may investigate any problem if you feel able to do so, in which
case the test suite provides a good starting point.  Its output may
be found below `rpmtests.dir'.

```

**Comment by pmatilai** [comment]
Yep, fixed by #2263 now, inspired by looking into the ultimately unrelated test-suite issues you were having. So it was actually a rather useful thing :grin:

**Comment by pmatilai** [comment]
Unfortunately find-debuginfo is not available on Debian so any debuginfo-related tests are being skipped, but this is much better than nothing at all.
